src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -151,10 +151,74 @@ public static class CPythonEvaluator
         private const string NodeName = "__dynamonodename__";
         static PyScope globalScope;
         internal static readonly string globalScopeName = "global";
+        private static DynamoLogger dynamoLogger;
+        private static IntPtr ts = new IntPtr();
```

Comment:
because we're no longer calling endAllowThreads should be able to make this local variable again.